test_cases/trench_test/morphological_hydro_fns.py

```diff
@@ -134,7 +133,7 @@ def update_forcings_hydrodynamics(t_new):
 
 def morphological(boundary_conditions_fn, morfac, morfac_transport, suspendedload, convectivevel, bedload, angle_correction, slope_eff, seccurrent, sediment_slide,
                   mesh2d, bathymetry_2d, input_dir, viscosity_hydro, ks, average_size, dt, final_time,
-                  beta_fn, surbeta2_fn, alpha_secc_fn, angle_fn, mesh_step_size, friction='nikuradse', friction_coef=0, d90=0, fluc_bcs=False, bed_form='meyer', sus_form='vanrijn', diffusivity=0.15):
+                  beta_fn=1.3, surbeta2_fn=1/1.5, alpha_secc_fn=0.75, angle_fn=35, mesh_step_size=0.2, friction='nikuradse', friction_coef=0, d90=0, fluc_bcs=False, bed_form='meyer', sus_form='vanrijn', diffusivity=0.15):
```

Comment:
Personally I prefer having long lists of args and kwargs for functions and prefer keeping them in a parameter class. This could either be an `Options` object, or just a dictionary `kwargs` (say) which you then pass as `**kwargs`.

Reply:
I agree it would be better to use options. This file probably shouldn't be included in this repo permanently - this is the file that I use for the normal thetis framework. I just included it so that I can test that adapt utils is doing the basic code right

Reply:
Fair doos!